0.13 docs update: tutorials (#4127)
garden-io/garden: docs/tutorials/your-first-project/3-deploy-and-test.md

garden deploy
```

You should see your applications come up. Garden will print the statuses of the corresponding `Deploy` actions. Next we
can set up our first test. Similar to how you configured the `Deploy` actions earlier, open the `frontend/garden.yml`
config and add the following action configurations:

```yaml

--- # the yaml separator is necessary to delimit different actions

kind: Test
name: frontend-unit
type: container
build: frontend
spec:
  args: [ npm, test ]

> --- # the yaml separator is necessary to delimit different actions

Reviewer: No need to repeat the comment imo, just distracts from other more important ones

Author: Removed in https://github.com/garden-io/garden/pull/4127/commits/e1230b742ace72516a30c9a2565fc9062f184f0a

kind: Test
name: frontend-integ
type: container
build: frontend
dependencies:

>   - deploy.frontend # <- we want the frontend service to be running and up-to-date for this test

Reviewer: Actually there's no need for this, but it can be reworded to something like

```suggestion
  - deploy.frontend # <- have the frontend service be running and up-to-date before the test
```
